src/main/java/Duke.java

```diff
@@ -1,10 +1,164 @@
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    public static String chatbotName = "Noriaki";
+
+    /**
+     * Prints line of 30 underscores.
+     */
+    public static void printLine(){
+        int lineUnderscore = 30;
+        for(int i = 0; i < lineUnderscore; i++){
+            System.out.print("_");
+        }
+        System.out.println();
+    }
+
+    /**
+     * Prints line, followed by message, followed by line.
+     *
+     * @param message Message to be printed.
+     */
+    public static void printMessage(String message){
+        printLine();
+        System.out.println(message);
+        printLine();
+    }
+
+    /**
+     * Prints greeting.
+     */
+    public static void greet(){
+        String greetMessage = "Hello! I'm " + chatbotName + "\nWhat can I do for you?";
+        String logo =
+                " _______               .__        __   .__ \n" +
+                " \\      \\   ___________|__|____  |  | _|__|\n" +
+                " /   |   \\ /  _ \\_  __ \\  \\__  \\ |  |/ /  |\n" +
+                "/    |    (  <_> )  | \\/  |/ __ \\|    <|  |\n" +
+                "\\____|__  /\\____/|__|  |__(____  /__|_ \\__|\n" +
+                "        \\/                     \\/     \\/   \n";
+
         System.out.println("Hello from\n" + logo);
+        printMessage(greetMessage);
+    }
+
+    /**
+     * Prints goodbye message.
+     */
+    public static void goodbye(){
```

Comment:
Names representing methods must be verbs and written in camelCase.
i think it would be better to name it as prinntGoodbyeMessage